packages/lambdas/src/cw-doc-contribution.ts

```diff
@@ -14,26 +15,27 @@ const region = getEnvOrFail("AWS_REGION");
 
 const s3Utils = new S3Utils(region);
 const bucketName = getEnvOrFail("MEDICAL_DOCUMENTS_BUCKET_NAME");
-const SIGNED_URL_DURATION_SECONDS = 60;
 
 /**
  * This lambda is called by CommonWell as part of document retrieval - DR.
  *
- * It's called after document query - DQ - with the Document Refeference's
+ * It's called after document query - DQ - with the Document Reference's
  * attachment URL - which points to here.
  *
  * This lambda should be behind API GW's OAuth authorizer at all times.
  *
  * It will:
- * - receive the attachment URL;
- * - generate a signed URL for the file (60s expiration);
- * - returns a redirect to the signed URL.
+ * - receive the fileName query parameter;
+ * - parse the filename to extract document ID and content type;
+ * - retrieve the document content from S3;
+ * - return a FHIR Binary resource with the document data.
  */
-// TODO move to capture.wrapHandler()
-export const handler = Sentry.AWSLambda.wrapHandler(
+const { log } = out("cw-doc-contribution");
```

Comment:
nit: let's keep those inside the function as much as possible - [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?source=copy_link#10eb51bb9040806fa1deeeffd3c811ac)